tests/local/missing-features/issue540.wast

```diff
@@ -0,0 +1,27 @@
+(module binary
+    "\00\61\73\6d"
+    "\01\00\00\00"
+
+    "\01\01\00"
+    "\02\01\00"
+
+    "\05\06\01"
+    "\01\70\e0\ab\02"
+
+    "\0b\08\01"
+    "\02\00\41\ca\0f"
+    "\0b\00")
+
+(module binary
+    "\00\61\73\6d"
+    "\01\00\00\00"
+
+    "\01\01\00"
+    "\02\01\00"
+
+    "\05\06\01"
+    "\01\70\e0\ab\02"
+
+    "\0b\08\01"
+    "\80\00\41\ca\0f"
+    "\0b\00")
```

Comment:
Could you add some comments to the side about what each of these binary segments mean? (there's some other `module binary` tests with examples of how this is done elsewhere.